pandas/tests/resample/conftest.py

```diff
@@ -63,7 +64,10 @@ def simple_period_range_series():
     """
 
     def _simple_period_range_series(start, end, freq="D"):
-        rng = period_range(start, end, freq=freq)
+        with warnings.catch_warnings():
+            # suppress Period[B] deprecation warning
+            warnings.simplefilter("ignore")
```

Comment:
could we be more precise with the `simplefilter` please? I think this may set a bad example for new contributors wanting to silence warnings and looking for existing patterns of how to do it

Reply:
good point, will update